### PR TITLE
Fix composer.json

### PR DESCRIPTION
**Pull request by michaelcullum**
Apply changes for new composer.json

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,32 +1,30 @@
 {
 	"name": "nickvergessen/newspage",
-	"type": "phpbb3-extension",
+	"type": "phpbb-extension",
 	"description": "Adds a extra-page to the board where a switchable number of news are displayed. The text can be shorten to a certain number of chars. Also the Icons can be switched of (post icons, user icons)",
 	"homepage": "https://github.com/nickvergessen/phpbb3-mod-newspage",
 	"version": "1.1.0",
-	"time": "2013-03-16 01:01:01",
+	"time": "2013-03-16",
```
Comment (michaelcullum) [inline composer.json:7]: Time part of `time` is optional

File: composer.json [hunk 2]
```diff
@@ -1,32 +1,30 @@
...
 	"licence": "GPL-2.0",
 	"authors": [
 		{
 			"name": "Joas Schilling",
-			"username": "nickvergessen",
```
Comment (michaelcullum) [inline composer.json:12]: No usernames

File: composer.json [hunk 3]
```diff
@@ -1,32 +1,30 @@
...
 			"homepage": "http://www.flying-bits.org",
 			"role": "Lead Developer"
 		},
 		{
 			"name": "Falk Seidel",
-			"username": "Unimatrix_0",
 			"email": "admin@un1matr1x.de",
 			"homepage": "http://un1matr1x.de",
-			"role": "Developer (since 2011-01-22)"
+			"role": "Developer"
```
Comment (michaelcullum) [inline composer.json:20]: No brackets or numbers or dates.

File: composer.json [hunk 4]
```diff
@@ -1,32 +1,30 @@
...
 		}
 	],
 	"require": {
 		"php": ">=5.3",
-		"phpbb": ">=3.1.0-dev"
+		"phpbb/phpbb": "3.1.*@dev"
```
Comment (nickvergessen) [inline composer.json:25]: It does work with A1, B1 and stable now though?

Comment (michaelcullum) [inline composer.json:25]: Yes? And? This is minimum stability.

**Pull request merged**